client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -3,23 +3,87 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import map from 'lodash';
+import Button from 'components/button';
+import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
 
 class JetpackOnboardingSummaryStep extends React.PureComponent {
+	renderCompleted = () => {
+		const { translate } = this.props;
+		const stepsCompleted = [
+			translate( 'Site title & description' ),
+			translate( 'Type of Site' ),
+			translate( 'Type of Homepage' ),
+			translate( 'Contact Us Form' ),
+			translate( 'Jetpack Connection' ),
+		];
+
+		return Object.keys( stepsCompleted ).map( stepCompleted => (
+			<div className="steps__summary-column entry completed">
+				<Gridicon icon="chevron-down" size={ 12 } />
+				{ stepsCompleted[ stepCompleted ] }
+			</div>
+		) );
+	};
+
+	renderTodo = () => {
+		const { translate } = this.props;
+		const stepsTodo = [
+			translate( 'Choose a Theme' ),
+			translate( 'Add a Site Address' ),
+			translate( 'Add a Store' ),
+			translate( 'Start a Blog' ),
+		];
+
+		// TODO: adapt when we have more info
+		const siteRedirectHref = '#';
+
+		return Object.keys( stepsTodo ).map( stepTodo => (
+			<div className="steps__summary-column entry todo">
+				<a href={ siteRedirectHref }>{ stepsTodo[ stepTodo ] }</a>
+			</div>
+		) );
+	};
+
 	render() {
 		const { translate } = this.props;
 		const headerText = translate( 'Congratulations! Your site is on its way.' );
 		const subHeaderText = translate(
 			'You enabled Jetpack and unlocked dozens of website-bolstering features. Continue preparing your site below.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		// TODO: adapt when we have more info
+		const buttonRedirectHref = '#';
+
+		return (
+			<Fragment>
+				<DocumentHead title={ translate( 'Summary ‹ Jetpack Onboarding' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Homepage ‹ Jetpack Onboarding' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-onboarding-summary-include-content)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).